presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java

```diff
@@ -101,6 +113,10 @@
 
     private final Ticker ticker;
 
+    private final boolean interruptRunawaySplits;
+    private final Predicate<List<StackTraceElement>> interruptibleSplitPredicate;
+    private final long interruptSplitInterval;
```

Comment:
`interruptSplitIntervalInSeconds` (or maybe simply use Duration)

Reply:
Switch to a duration